pkg/controller/complianceremediation/complianceremediation_controller.go

```diff
@@ -236,23 +239,100 @@ func (r *ReconcileComplianceRemediation) reconcileRemediation(instance *compv1al
 				"Make sure the CRD is installed: %w", err)
 	} else if kerrors.IsNotFound(err) {
 		if instance.Spec.Apply {
+			// Going through remediation list, to make sure all the related
+			// remediations objects are set to apply
+			err := r.setRemediationsToApply(instance, objectLogger)
+			if err != nil {
+				objectLogger.Error(err, "Failed to set related remediations to apply")
+				return err
```

Comment:
can we return wrapped errors?
e.g. `return fmt.Errorf("Cannot apply related remediation: %w, err")`